ote/src/cljs/ote/ui/buttons.cljs

```diff
@@ -27,3 +29,11 @@
                     :font-size "12px"
                     :font-weight "bold"}} opts)
     label]])
+
+(defn open-link [e! url]
```

Comment:
could we use the common `linkify` utility instead of this new button?

Reply:
Yes, please.

Reply:
We could, but ui spec says that it must be a flat-button

Reply:
Well, you could do it like this:
```cljs
[ui/flat-button
  {:label (r/as-element 
               (common/linkify "http://nonsense" "Go!" {:target "_blank"}))
  :primary true}]
```

Reply:
That won't work. You can press the button and nothing happens if mouse pointer is not on top of label.

Reply:
It turns out, you can actually do it like this:
```cljs
 (common/linkify "http://nonsense" 
  [ui/flat-button {:label "Go! :primary true}]
   {:target "_blank"})
```
The a-element will properly wrap the "button" element and the whole button is then interpreted as a link.

Reply:
This is why I love these reviews. Thanx! That linkify works marvellously.
Changes coming up.